### PR TITLE
print proxy information if it exists

### PR DESCRIPTION
**Pull request by mdubbyap**

### CODE REVIEW

File: protocol/signalfx/signalfxforwarder.go [hunk 1]
```diff
@@ -112,6 +112,15 @@ func NewSignalfxJSONForwarer(url string, timeout time.Duration,
 		},
 		TLSHandshakeTimeout: timeout,
 	}
+
+	req, _ := http.NewRequest("POST", "https://www.signalfx.com", nil)
```
Comment (cep21) [inline protocol/signalfx/signalfxforwarder.go]: Should you use the URL that is being forwarded to?

Comment (mdubbyap) [inline protocol/signalfx/signalfxforwarder.go]: yeah...

File: protocol/signalfx/signalfxforwarder.go [hunk 2]
```diff
@@ -112,6 +112,15 @@ func NewSignalfxJSONForwarer(url string, timeout time.Duration,
...
+	if req != nil {
+		purl, _ := tr.Proxy(req)
```
Comment (cep21) [inline protocol/signalfx/signalfxforwarder.go]: Can you also print something if you get an error?  It tells people if their PROXY env is malformed

Comment (mdubbyap) [inline protocol/signalfx/signalfxforwarder.go]: yup, done.

File: protocol/signalfx/signalfxforwarder.go [hunk 3]
```diff
@@ -112,6 +112,17 @@ func NewSignalfxJSONForwarer(url string, timeout time.Duration,
 		},
 		TLSHandshakeTimeout: timeout,
 	}
+
+	req, _ := http.NewRequest("POST", url, nil)
+	if req != nil {
+		purl, err := tr.Proxy(req)
```
Comment (cep21) [inline protocol/signalfx/signalfxforwarder.go]: If you want to test this, create a function similar to this:

```
func logProxyURL(t *http.Transport) (wasLogged bool) {
// Put your code here
}
```

You can then call that with a transport that has a Proxy that returns false, and another that returns true, and test if wasLogged.

**Pull request closed without merging**